health/micro-ui/web/core/Dockerfile

```
@@ -1,24 +1,38 @@
-    &&  ./install-deps.sh \
-    && cd ../ \
-    && yarn install \ 
-    && yarn build:webpack 
+# Set working directory
+WORKDIR /app/web
+
+# Show heap size limit
+RUN node -e "console.log('Heap limit (MB):', v8.getHeapStatistics().heap_size_limit / (1024 * 1024))"
+
+# Print a marker log (optional debug message)
+RUN node -e "console.log('health-core only')"
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)